deps.sh

```diff
@@ -583,15 +583,29 @@ fi
 
 if [[ $INSTALL_OPENVINO == true ]] && [[ ! -f $BUILD_DIR/openvino.done ]] ; then
     echo "Installing OpenVINO Inference Engine 2019 R3"
+    OPENVINO_REQ_PKGS=(
+        libusb-1.0-0-dev
+        libgstreamer1.0-0
+        gstreamer1.0-plugins-base
+        gstreamer1.0-plugins-good
+        gstreamer1.0-plugins-bad
+    )
+    apt update
+    apt install -y ${OPENVINO_REQ_PKGS[@]}
```

Comment:
This command might fail if the user isn't sudo. Any thoughts on how to deal with that?

Reply:
Some thoughts: 
1. We can move these dependencies to ubuntu16.04/Dockerfile.base. That way, users have these dependencies already installed in the Docker image, or when following the Dockerfile to create their staging machine. 
2. We can have an additional .sh file, e.g., deps_openvino.sh, with these dependencies. If users want to use OpenVINO, they should also execute that script. We can add a message about it at the end of deps.sh. 
3. We assume that deps.sh is executed with sudo (sudo bash ./deps.sh -a --prefix /usr/local) since installing in /usr/local requires sudo. This is a weak assumption since it fails if they want to use another location.

Reply:
Both options 1 and 2 sound good to me. Perhaps both? That is, we create the `deps_openvino.sh` script -- which can be run if someone wants to install OpenVino -- and then we can execute the `deps_openvino.sh` script inside the Dockerfile to install those dependencies in the image.

Reply:
Done! deps_openvino.sh script has been created. Users will have to run this script if they plan to use OpenVINO. This script is executed inside the Dockerfile before deps.sh. Thanks!